.goreleaser.yaml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+# before:
+#   hooks:
+    # You may remove this if you don't use go modules.
+    # - go mod tidy
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    main: ./cmd/golangci-lint-kube-api-linter/
+
+archives:
+  - formats: [tar.gz]
+    # this name template makes the OS and Arch compatible with the results of `uname`.
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    # use zip for windows archives
+    format_overrides:
+      - goos: windows
+        formats: [zip]
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
```

Comment:
Out of scope for this PR, but this makes me think we should probably start enforcing a commit formatting convention so we can more reliably remove things we think aren't applicable to include in the changelogs.

Probably also just good hygiene anyways :shrug:

Reply:
Yeah I'll check what others are doing in this regard, I think CAPI/controller-runtime both have a fairly reasonable pattern